reporters/kamon-prometheus/src/main/scala/kamon/prometheus/embeddedhttp/EmbeddedHttpServer.scala

```diff
@@ -19,6 +19,6 @@ package kamon.prometheus.embeddedhttp
 import com.typesafe.config.Config
 import kamon.prometheus.ScrapeSource
 
-abstract class EmbeddedHttpServer(hostname: String, port: Int, scrapeSource: ScrapeSource, config: Config) {
+abstract class EmbeddedHttpServer(hostname: String, port: Int, path: String, scrapeSource: ScrapeSource, config: Config) {
```

Comment:
will this not break backwards compatibility for users that already defined their own `EmbeddedHttpServer` ?

Reply:
I was not aware that people extended that! Fixed!